diff -Nur griffith-0.9~rc1.org/lib/gutils.py griffith-0.9~rc1/lib/gutils.py
--- griffith-0.9~rc1.org/lib/gutils.py	2006-12-28 16:41:04.000000000 +0100
+++ griffith-0.9~rc1/lib/gutils.py	2006-12-28 16:43:53.000000000 +0100
@@ -301,7 +301,7 @@
 def read_plugins(prefix,directory):
 	"""returns available plugins"""
 	import glob
-	return glob.glob("%s/%s*.py" % (directory,prefix) )
+	return glob.glob("%s/%s*.pyc" % (directory,prefix) )
 
 def findKey(val, dict):
 	for key, value in dict.items():
diff -Nur griffith-0.9~rc1.org/lib/initialize.py griffith-0.9~rc1/lib/initialize.py
--- griffith-0.9~rc1.org/lib/initialize.py	2006-12-28 16:41:04.000000000 +0100
+++ griffith-0.9~rc1/lib/initialize.py	2006-12-28 16:45:19.000000000 +0100
@@ -306,7 +306,7 @@
 	self.d_plugin = 0
 	mcounter = 0
 	for p in self.plugins:
-		plugin_module = os.path.basename(p).replace('.py','')
+		plugin_module = os.path.basename(p).replace('.pyc','')
 		plugin_name = plugin_module.replace('PluginMovie','')
 		self.widgets['add']['source'].append_text(plugin_name)
 		self.widgets['preferences']['default_plugin'].append_text(plugin_name)
@@ -325,7 +325,7 @@
 		self.locations['export_plugins'])
 	plugins.sort()
 	for p in plugins:
-		plugin_module = os.path.basename(p).replace('.py', '')
+		plugin_module = os.path.basename(p).replace('.pyc', '')
 		plugin_name = plugin_module.replace('PluginExport', '')
 		menu_items = gtk.MenuItem(plugin_name)
 		self.widgets['menu']['export'].append(menu_items)
