--- griffith-0.11/lib/gutils.py~	2010-01-09 23:45:18.000000000 +0100
+++ griffith-0.11/lib/gutils.py	2010-01-19 13:08:32.234490934 +0100
@@ -390,7 +390,7 @@
     """returns available plugins"""
 
     import glob
-    return glob.glob("%s/%s*.py" % (directory, prefix))
+    return glob.glob("%s/%s*.pyc" % (directory, prefix))
 
 
 def findKey(val, dict):
--- griffith-0.11/lib/initialize.py~	2010-01-05 21:06:06.000000000 +0100
+++ griffith-0.11/lib/initialize.py	2010-01-19 13:10:01.041150494 +0100
@@ -423,7 +423,7 @@
     mcounter = 0
     default_plugin = self.config.get('default_movie_plugin')
     for p in self.plugins:
-        plugin_module = os.path.basename(p).replace('.py', '')
+        plugin_module = os.path.basename(p).replace('.pyc', '')
         plugin_name = plugin_module.replace('PluginMovie', '')
         self.widgets['add']['source'].append_text(plugin_name)
         self.widgets['preferences']['default_plugin'].append_text(plugin_name)
@@ -442,7 +442,7 @@
         self.locations['export_plugins'])
     plugins.sort()
     for p in plugins:
-        plugin_module = os.path.basename(p).replace('.py', '')
+        plugin_module = os.path.basename(p).replace('.pyc', '')
         plugin_name = plugin_module.replace('PluginExport', '')
         menu_items = gtk.MenuItem(plugin_name)
         self.widgets['menu']['export'].append(menu_items)
--- griffith-0.13/lib/plugins/imp/__init__.py~	2011-01-18 12:25:50.000000000 +0200
+++ griffith-0.13/lib/plugins/imp/__init__.py	2011-12-26 23:29:26.563009107 +0200
@@ -36,7 +36,7 @@
 import edit
 
 # detect all plugins:
-__all__ = [os.path.basename(x)[:-3] for x in glob.glob("%s/*.py" % os.path.dirname(__file__))]
+__all__ = [os.path.basename(x)[:-4] for x in glob.glob("%s/*.pyc" % os.path.dirname(__file__))]
 __all__.remove('__init__')
 
 
--- griffith-0.13/lib/plugins/export/__init__.py~	2011-02-08 23:10:58.000000000 +0200
+++ griffith-0.13/lib/plugins/export/__init__.py	2011-12-26 23:30:08.681616820 +0200
@@ -36,7 +36,7 @@
 from sql import update_whereclause
 
 # detect all plugins:
-__all__ = [os.path.basename(x)[:-3] for x in glob.glob("%s/PluginExport*.py" % os.path.dirname(__file__))]
+__all__ = [os.path.basename(x)[:-4] for x in glob.glob("%s/PluginExport*.pyc" % os.path.dirname(__file__))]
 
 class Base(object):
     description = None
