diff -Nur griffith-0.9.org/lib/gutils.py griffith-0.9/lib/gutils.py
--- griffith-0.9.org/lib/gutils.py	2007-02-20 21:27:11.000000000 +0100
+++ griffith-0.9/lib/gutils.py	2007-02-20 21:29:27.000000000 +0100
@@ -299,7 +299,7 @@
 def read_plugins(prefix,directory):
 	"""returns available plugins"""
 	import glob
-	return glob.glob("%s/%s*.py" % (directory,prefix) )
+	return glob.glob("%s/%s*.pyc" % (directory,prefix) )
 
 def findKey(val, dict):
 	for key, value in dict.items():
diff -Nur griffith-0.9.org/lib/initialize.py griffith-0.9/lib/initialize.py
--- griffith-0.9.org/lib/initialize.py	2007-02-20 21:27:11.000000000 +0100
+++ griffith-0.9/lib/initialize.py	2007-02-20 21:29:50.000000000 +0100
@@ -313,7 +313,7 @@
 	self.d_plugin = 0
 	mcounter = 0
 	for p in self.plugins:
-		plugin_module = os.path.basename(p).replace('.py','')
+		plugin_module = os.path.basename(p).replace('.pyc','')
 		plugin_name = plugin_module.replace('PluginMovie','')
 		self.widgets['add']['source'].append_text(plugin_name)
 		self.widgets['preferences']['default_plugin'].append_text(plugin_name)
@@ -332,7 +332,7 @@
 		self.locations['export_plugins'])
 	plugins.sort()
 	for p in plugins:
-		plugin_module = os.path.basename(p).replace('.py', '')
+		plugin_module = os.path.basename(p).replace('.pyc', '')
 		plugin_name = plugin_module.replace('PluginExport', '')
 		menu_items = gtk.MenuItem(plugin_name)
 		self.widgets['menu']['export'].append(menu_items)
