diff -Nur griffith-0.6.1.org/lib/gutils.py griffith-0.6.1/lib/gutils.py
--- griffith-0.6.1.org/lib/gutils.py	2006-05-28 23:51:16.000000000 +0200
+++ griffith-0.6.1/lib/gutils.py	2006-05-28 23:54:34.000000000 +0200
@@ -353,7 +353,7 @@
 def read_plugins(prefix,directory):
 	"""returns available plugins"""
 	import glob
-	return glob.glob("%s/%s*.py" % (directory,prefix) )
+	return glob.glob("%s/%s*.pyc" % (directory,prefix) )
 
 def findKey(val, dict):
 	for key, value in dict.items():
diff -Nur griffith-0.6.1.org/lib/initialize.py griffith-0.6.1/lib/initialize.py
--- griffith-0.6.1.org/lib/initialize.py	2006-05-28 23:50:45.000000000 +0200
+++ griffith-0.6.1/lib/initialize.py	2006-05-28 23:55:57.000000000 +0200
@@ -203,7 +203,7 @@
 	self.d_plugin = 0
 	mcounter = 0
 	for p in self.plugins:
-		plugin_module = os.path.basename(p).replace(".py","")
+		plugin_module = os.path.basename(p).replace(".pyc","")
 		plugin_name = plugin_module.replace("PluginMovie","")
 		self.am_source.append_text(plugin_name)
 		self.default_plugin.append_text(plugin_name)
@@ -222,7 +222,7 @@
 		self.locations['export_plugins'])
 	plugins.sort()
 	for p in plugins:
-		plugin_module = os.path.basename(p).replace(".py", "")
+		plugin_module = os.path.basename(p).replace(".pyc", "")
 		plugin_name = plugin_module.replace("PluginExport", "")
 		menu_items = gtk.MenuItem(plugin_name)
 		self.export_menu.append(menu_items)
